apps/namadillo/src/types.d.ts

```diff
@@ -189,13 +189,16 @@ export type ChainRegistryEntry = {
   ibc: IBCInfo[];
 };
 
-export type AddressWithAssetAndBalance = {
-  address: string;
+export type AddressWithAsset = {
```

Comment:
To simplify some names, I think we could call this as `Token`, wdyt?

My reference is the `tokens.toml` files that is used by the cli, that is basically the `tnam...` address plus some properties

Reply:
I definitely agree the names I used are not very good. I'm not sure about `Token` since I think we have other things named similar to Token (but maybe those are not needed now, I have to check). `AssetRegistry` makes me think more of the full registry, like cosmos/chain-registry.

For now, I created #1244 to track this and I think it should be fairly easy to change the names when we settle on something.

Reply:
Some ChatGPT recommendations:

1. AssetRecord (address and asset only) and AssetAmountRecord (address, asset, and amount)
2. AssetReference (address and asset only) and AssetAmountEntry (address, asset, and amount)
3. ResolvedAsset (address and asset only) and ResolvedAssetAmount (address, asset, and amount)
4. TokenEntry (address and asset only) and TokenAmountEntry (address, asset, and amount)

I'm not sure about these since they don't indicate that there is an original address, but maybe they're OK.